src/pages/home/HomePage.js

```diff
@@ -112,18 +115,21 @@ class HomePage extends PureComponent {
                     <ReportScreen />
 
                     {/* Modal Screens */}
-                    <RightDockedModal route={ROUTES.SETTINGS}>
+                    <RightDockedModal routes={[ROUTES.SETTINGS]}>
                         <SettingsPage />
                     </RightDockedModal>
-                    <RightDockedModal route={ROUTES.NEW_GROUP}>
+                    <RightDockedModal routes={[ROUTES.NEW_GROUP]}>
                         <NewGroupPage />
                     </RightDockedModal>
-                    <RightDockedModal route={ROUTES.NEW_CHAT}>
+                    <RightDockedModal routes={[ROUTES.NEW_CHAT]}>
                         <NewChatPage />
                     </RightDockedModal>
-                    <RightDockedModal route={ROUTES.SEARCH}>
+                    <RightDockedModal routes={[ROUTES.SEARCH]}>
                         <SearchPage />
                     </RightDockedModal>
+                    <RightDockedModal routes={[ROUTES.IOU_REQUEST_MONEY, ROUTES.IOU_BILL_SPLIT]}>
```

Comment:
I think maybe it would be best to avoid changing this to `routes`. If we need the `IOUModal` to show up for both of these routes then we can just have it twice for now.

Reply:
That's how I originally set this up. Here's [the comment](https://github.com/Expensify/Expensify.cash/pull/1561#discussion_r587750355) that led to that change for additional context.

Reply:
Thanks for sharing, looks like I might need to revert in that case.

Reply:
Cool. I _think_ we had it right the first time. It would be great to not have any `Route` components apart from the ones on the `Homepage` as it will make transitioning to `react-navigation` easier.